client/src/components/HackathonForm.jsx

```diff
@@ -0,0 +1,163 @@
+            season: "",
+            categories: [{
+              name: "",
+              companyName: "",
+              type: "",
+            }],
+          }}
+          validationSchema={validationSchema}
+          onSubmit={(values) => {
+            alert(JSON.stringify(values, null, 2));
+          }}
+        >
+          {({ values, handleChange, setFieldValue }) => (
+            <Form className={classes.root} autoComplete="off">
+              <TextField
+                name="season"
+                value={values.season}
+                onChange={handleChange}
+                label="Season"
+                helperText="Season of this Hackathon"
+                fullWidth
+                required
+              />
+
+              <FieldArray name="categories">
+                {({ push , remove }) => (
+                  <>
+                  <FormLabel className={classes.urlFieldLabel}>
+                    Categories
+                  </FormLabel>
+                  {values.categories.length === 0 ? (
+                  <FormLabel className={classes.urlFieldLabel}>
+                    Need At Least 1 Category!!!!!!
+                  </FormLabel>
+                ) : (
+                  <></>
+                )}
+                {/* eslint-disable-next-line */}
+                {values.categories.map((category, index) => {
+                  return (
+                    <div>
+                      <Grid
+                        container
+                        direction="row"
+                        justify="flex-start"
+                        alignItems="baseline"
+                      >
+
+                        <TextField
+                          className={classes.categoryField}
+                          name={`categories[${index}].name`}
+                          value={category.name}
+                          onChange={handleChange}
+                          label="Name"
+                          helperText="Name of the Category"
+                          required
+                        />
+
+                        <TextField
+                          className={classes.companyField}
+                          name={`categories[${index}].companyName`}
+                          value={category.companyName}
+                          onChange={handleChange}
+                          label="Company Name"
+                          helperText="Name of the company sponsoring the category (if Applicable)"
+                        />
+
+                        <TextField
+                          className={classes.typeField}
+                          name={`categories[${index}].type`}
+                          value={category.type}
+                          onChange={handleChange}
+                          label="Type"
+                          helperText="Type of the category, either 'Track' or 'Superlative'"
+                          required
+                        />
```

Comment:
Instead of a text field, this should be a select/dropdown menu, because the only two choices are track or superlative